{"uuid": "a5caa778-fd2d-48d2-90bb-cf8f31b81d3e", "children": ["1c859b6f-49c6-4402-9787-c964fc4b3686"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 80, in setup_profiles\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/fixtures_4x.py\", line 111, in setup_profiles\n    instantiate_profile_obj.check_and_set_ssid_sec(\"2G\", profile_data[\"ssid\"][mode][0][\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 670, in check_and_set_ssid_sec\n    self.get_all_ssid_detail(radio=radio)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 716, in get_all_ssid_detail\n    ssid_details = self.run_generic_cli_command(cmd)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 180, in run_generic_cli_command\n    output = os.popen(cmd).read()\n"}, "start": 1696930659757, "stop": 1696930661531}], "start": 1696930659757, "stop": 1696930661716}